contracts/BatchExchange.sol

```diff
@@ -212,7 +207,7 @@ contract BatchExchange is EpochTokenLocker {
       *
       * @param indices referencing the index of user's order to be canceled
       *
-      * Emits an {OrderCancelation} or {OrderDeletion} with sender's address and orderIndex
+      * Emits an {OrderCancelation} or {OrderDeletion} with sender's address and orderId
       */
     function cancelOrders(uint256[] memory indices) public {
         for (uint256 i = 0; i < indices.length; i++) {
```

Comment:
`indices` should be a `uint16` and follow the same naming convention as the other places.

Reply:
Are we sure this doesn't need to be u256 becaus eits going to be used in an array access?

Reply:
It should be ID anyway right?

Reply:
We do array access with `uint16` in other places so I imagine its OK. Also it should be `orderIds` or `orderIndices`.

Reply:
I'll use `orderIds`.  I can see where you are coming from, and I do really like this suggestion (since it is not truly a unique identifier...) but I somewhat prefer the conciseness of orderIds.